### PR TITLE
Compilation: Misc changes to support XBuild

### PR DESCRIPTION
**Pull request by larsjohnsen**
See #210

### CODE REVIEW

File: src/NzbDrone.Core/NzbDrone.Core.csproj [hunk 1]
```diff
@@ -970,6 +970,10 @@
       <Link>MediaInfo.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\MediaInfoDotNet.dll.config">
+      <Link>MediaInfoDotNet.dll.config</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```
Comment (larsjohnsen) [inline src/NzbDrone.Core/NzbDrone.Core.csproj]: Note that this is currently done here: https://github.com/Sonarr/Sonarr/blob/develop/build.ps1#L117

**Comment by markus101** [comment]
@Taloth this looks good to me too (and I manually ran a build on TeamCity, which passed).

File: src/NzbDrone.Core/NzbDrone.Core.csproj (hunk 1 above)
Comment (Taloth) [inline src/NzbDrone.Core/NzbDrone.Core.csproj]: @markus101 Yes, but this part has to be removed, was changed to NzbDrone.Core.dll.config in the mediainfo fix.

**Comment by larsjohnsen** [comment]
Pull request updated to reflect changes in 5cd2d71e6fd51a36948b24fb029a0ae032b08a74.

**Pull request merged**